src/Authentication/AppBuilderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace EWS.Authentication
+{
```

Comment:
PP: Generell braucht es nach Namespaces keine geschweiften Klammern mehr, d.h. auch keine Einrückung (siehe andere Klassen im Projekt). Kannst es aber drin lassen und ggf. in einem separaten PR bereinigen, falls du willst.